orchagent/intfsorch.cpp

```diff
@@ -57,6 +57,21 @@ static const vector<sai_router_interface_stat_t> rifStatIds =
     SAI_ROUTER_INTERFACE_STAT_OUT_ERROR_OCTETS,
 };
 
+// Translation of loopback action from string to sai type
+const unordered_map<string, loopback_action_e> IntfsOrch::m_loopback_action_map =
+{
+    {"none",    LOOPBACK_ACTION_NONE},
```

Comment:
Please remove "none" if it is not one of the available options for the user. I believe LOOPBACK_ACTION_NONE is used only internally within orchagent during intf creation.

Reply:
Done.